ceph-build/config/definitions/ceph-build.yml

```diff
@@ -10,7 +10,7 @@
       - github:
           url: https://github.com/ceph/ceph
     execution-strategy:
-       combination-filter: DIST==AVAILABLE_DIST && ARCH==AVAILABLE_ARCH && (ARCH=="x86_64" || (ARCH == "arm64" && (DIST == "xenial" || DIST == "centos7")))
+       combination-filter: DIST == AVAILABLE_DIST && ARCH == AVAILABLE_ARCH && (ARCH=="x86_64" || (ARCH == "arm64" && ["xenial", "centos7"].contains(DIST))) && (${GIT_BRANCH, fullName=false} ==~ /(jewel|luminous).*/ || (! ${GIT_BRANCH, fullName=false} ==~ /(jewel|luminous).*/ && DIST != "trusty")))
```

Comment:
have you tested any of this? this looks super scary

Reply:
@alfredodeza i tested using my own jenkins instance. but i don't have the necessary privilege for testing my change on jenkins.ceph.com , so i haven't test this PR using it yet. i mailed you my API token in hope that i will be able to test it.